Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ContextFactory.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fakebook.Profile.DataAccess;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+namespace Fakebook.Profile.UnitTests.APITests
```

Comment:
naming convention would have this namespace and folder be `ApiTests`.

Reply:
Renamed folder and all namespaces